api/coreservice.go

```diff
@@ -491,7 +491,21 @@ func (core *coreService) SendAction(ctx context.Context, in *iotextypes.Action)
 }
 
 func (core *coreService) PendingNonce(addr address.Address) (uint64, error) {
-	return core.ap.GetPendingNonce(addr.String())
+	nonce, err := core.ap.GetPendingNonce(addr.String())
+	if err != nil {
+		return nonce, err
+	}
+	g := core.Genesis()
+	if nonce > 1 || !g.IsToBeEnabled(core.TipHeight()) {
```

Comment:
nonce returned in line 494 is from `ap`. if `nonce == 1 && after hard fork`, shall we return 1?

Reply:
fixed in latest commit